app_feup/lib/view/Widgets/bug_report_form.dart

```diff
@@ -91,16 +89,29 @@ class BugReportFormState extends State<BugReportForm> {
       bottomMargin: 30.0,
     ));
 
+    formWidget.add(FormTextField(
+      emailController,
```

Comment:
Should we add email validation here?

Reply:
I think a simple regex validator would do. Should I extend the FormFieldClass to override the validator or perhaps include as a parameter in the existing class?

Reply:
I think there's already support for validators, but using a TextFormField instead of a FormTextField? I don't get it either ahah. Take a look at this: https://stackoverflow.com/a/63293006

Reply:
My question was what's the best design, pass validateEmail as a new parameter to FormTextField and check if it's set in the validator or extend FormTextField to something like EmailFormTextField to override the validator. Also, would it be worth to add a new package, giving the regex check is two lines (defining and checking for a match)?

Reply:
Regarding the last question, I took a look at the source code of `email_validator`and it is indeed small, I'm including it

Reply:
Yes using that package should not be a problem, but my suggestion was regarding the design too. It seems that the **TextFormField** they use already has a "validator" parameter which makes it easier to maintain and I believe also gives [UI feedback to the user](https://api.flutter.dev/flutter/material/TextFormField-class.html). If you didn't use that package you could create your own validator separately and use it there, but it's already very small so why not xD

Reply:
In the entire bug report page we're using a custom `FormTextField` class which wraps `TextFormField` from the Material Design widgets. This is confusing naming schema to me, however I won't change it in the scope of this PR. This does imply however that the interface is a bit different and the validator behaviour depends on some parameters. To make the validator work for emails I added a new attribute. Please check it out